src/app/games/tafl/TaflMove.ts

```diff
@@ -0,0 +1,73 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { MoveCoordToCoord } from 'src/app/jscaip/MoveCoordToCoord';
+import { Direction } from 'src/app/jscaip/Direction';
+import { NumberEncoder } from 'src/app/jscaip/Encoder';
+
+export class TaflEncoder<M extends TaflMove> extends NumberEncoder<M> {
+
```

Comment:
my bad je croyais que c'était pas encore mergé mais ça l'est depuis un moment: please use NumberEncoder.tuple instead of reimplementing it by hand

Reply:
WELL DO IT YOURSELF MATE ! (haha, no, on it :D )